### PR TITLE
Fix docker network

### LINKED ISSUE #226: .NetworkSettings.IPAddress does not work for non-default Docker networks
```toml
[docker]
endpoint = "unix:///var/run/docker.sock"
domain = "docker.localhost"
watch = true

[web]
address = ":8080"
```

```
docker run -d -p 8080:8080 -p 80:80 -v /var/run/docker.sock:/var/run/docker.sock -v $PWD/traefik.toml:/traefik.toml containous/traefik
docker run swarm:1.0.0 manage token://blablabla
```

screen shot 2016-02-27 at 00 30 37

```
docker network create bla
docker run -d --net=bla -p 8080:8080 -p 80:80 -v /var/run/docker.sock:/var/run/docker.sock -v $PWD/traefik.toml:/traefik.toml containous/traefik
docker run --net=bla swarm:1.0.0 manage token://blablabla
```

screen shot 2016-02-27 at 00 30 18

This especially hurts docker-compose setups, as it creates a custom network by default.

### PR DESCRIPTION
**Pull request by emilevauge**
This PR fixes docker network management using `NetworkSettings.Networks` instead of `NetworkSettings.IPAddress`.
Fixes #226

### CODE REVIEW

File: templates/docker.tmpl [hunk 1]
```diff
@@ -1,6 +1,6 @@
 [backends]{{range .Containers}}
     [backends.backend-{{getBackend .}}.servers.server-{{.Name | replace "/" "" | replace "." "-"}}]
-    url = "{{getProtocol .}}://{{.NetworkSettings.IPAddress}}:{{getPort .}}"
+    url = "{{getProtocol .}}://{{range $i := .NetworkSettings.Networks}}{{if $i}}{{.IPAddress}}{{end}}{{end}}:{{getPort .}}"
```
Comment (vdemeester) [inline templates/docker.tmpl]: @emilevauge what if the container is connected to more than one network ? :stuck_out_tongue_closed_eyes:

Comment (emilevauge) [inline templates/docker.tmpl]: Yeah, we should introduce a new label `traefik.docker.network` that would default point to `bridge` if multiple networks detected. Right?

Comment (vdemeester) [inline templates/docker.tmpl]: I was thinking of something like that, maybe :wink:

**Comment by JeanMertz** [comment]
Thanks for the quick fix @emilevauge. I'll be sure to test this sometime this weekend :tada:

**Comment by vdemeester** [comment]
ping @emilevauge, want to handle the "multiple network" here or in a follow-up PR ? :angel:

**Comment by JeanMertz** [comment]
I've tested this implementation locally, works as advertised :+1: 

I'd say, ship it, and make it multi-network compatible in a new PR (the multi-network setup is a lot less common than the single-network one is).

**Comment by emilevauge** [comment]
I agree with @JeanMertz, let's merge this ;)
Created #229
/cc @vdemeester @rebaseTraefik

**Comment by rebaseTraefik** [comment]
I just pushed up the changes, enjoy!

**Comment by vdemeester** [comment]
LGTM 🐸 then :+1:

**Pull request merged**